src/errors/Messages.js

```diff
@@ -66,7 +66,7 @@ const Messages = {
   IMAGE_SIZE: size => `Invalid image size: ${size}`,
 
   MESSAGE_BULK_DELETE_TYPE: 'The messages must be an Array, Collection, or number.',
-  MESSAGE_NONCE_TYPE: 'Message nonce must fit in an unsigned 64-bit integer.',
+  MESSAGE_NONCE_TYPE: 'Message nonce must be a number or a string.',
```

Comment:
doesn't changing this error message count as a breaking change?

Reply:
I'm not really sure about it.

Reply:
IMO [`Error#code`](https://github.com/discordjs/discord.js/blob/master/src/errors/DJSError.js#L25) is the semver-relevant part about error messages, but that might be just me being optimistic.
In this case that would be `MESSAGE_NONCE_TYPE`.